ironfish/src/telemetry/interfaces/tag.ts

```diff
@@ -1,7 +1,7 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-export interface Tag {
+export type Tag = {
```

Comment:
nit: As discussed offline, what do you think about keeping this as an [interface](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces)?

Reply:
Reverted